s4e-backend/src/main/java/pl/cyfronet/s4e/controller/validation/GroupNameValidator.java

```diff
@@ -0,0 +1,28 @@
+package pl.cyfronet.s4e.controller.validation;
+
+import lombok.Data;
+import pl.cyfronet.s4e.service.InstitutionService;
+
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+
+@Data
+public class GroupNameValidator implements ConstraintValidator<GroupNameValid, String> {
+    private final InstitutionService institutionService;
+
+    @Override
+    public void initialize(GroupNameValid constraintAnnotation) {
+
+    }
+
+    @Override
+    public boolean isValid(String s, ConstraintValidatorContext constraintValidatorContext) {
+        if (s == null) return false;
+
+        if (s.contains(institutionService.PREFIX) ||
```

Comment:
Doesn't the first condition also cover the second one?
Also, the validator allows to add a group with name "default", which should be rejected too, doesn't it? It's slug is equal to 'default', and the unique index fails then.